.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           npm ci
           npx playwright install --with-deps
       - name: Run tests
-        run: npm run test:coverage
+        run: npm run test:all
```

Comment:
Doing this also fixed the coverage capture. Now the results are similar/better locally and pass the thresholds given.